src/golang/lib/notification/email.go

```diff
@@ -27,9 +29,30 @@ func (e *EmailNotification) Level() shared.NotificationLevel {
 	return e.conf.Level
 }
 
+func fullMessage(subject string, from string, targets []string, body string) string {
+	fullMsg := fmt.Sprintf("From: %s\n", from)
+	fullMsg += fmt.Sprintf("To: %s\n", strings.Join(targets, ","))
+	fullMsg += fmt.Sprintf("Subject: %s\n\n", subject)
+	fullMsg += body
+	return fullMsg
+}
+
 func (e *EmailNotification) Send(ctx context.Context, msg string) error {
-	// TODO: Implement
-	return nil
+	auth := smtp.PlainAuth(
+		"", // identity
+		e.conf.User,
+		e.conf.Password,
+		e.conf.Host,
+	)
+
+	fullMsg := fullMessage("aqueduct notification", e.conf.User, e.conf.Targets, msg)
```

Comment:
Should we change the subject based on the result of the notification?

Or is that out of scope here?

Reply:
Yeah we do plan to make it more dynamic. https://linear.app/aqueducthq/issue/ENG-2367/[m4]-add-proper-subject-for-email-notifications

I'd like to do it in a separate PR but it's def. something we want for release